### PR TITLE
Inspect 2.6 keycloak changes

### PR DESCRIPTION
**Pull request by davidclaveau**
Changelog:
- Update to look at Keycloak Gold standard realm

### CODE REVIEW

File: app/openshift/app.dc.yaml
Comment (Christopher-walsh22) [inline app/openshift/app.dc.yaml:1]: Should line #239 be changed to "inspect-bc-mussels-4817"

**Review by Christopher-walsh22 — APPROVED** [review]

File: env_config/env.docker [hunk 1]
```diff
@@ -44,8 +44,8 @@ DB_SCHEMA=invasivesbc
 # ------------------------------------------------------------------------------
 # KeyClock URLS
 # ------------------------------------------------------------------------------
-APP_CERTIFICATE_URL=https://dev.oidc.gov.bc.ca/auth/realms/dfmlcg7z/protocol/openid-connect/certs
-APP_CERTIFICATE_URL_TEST=https://oidc.gov.bc.ca/auth/realms/dfmlcg7z/protocol/openid-connect/certs
+APP_CERTIFICATE_URL=https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/certs
+APP_CERTIFICATE_URL_TEST=https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/certs
```
Comment (norrisng-bc) [inline env_config/env.docker]: Old value for `APP_CERTIFICATE_URL_TEST` points to Silver PROD. Is this supposed to point at Gold PROD as well?

File: loadTest/app/env.example [hunk 2]
```diff
@@ -2,6 +2,6 @@
 TEST_USER=#NAME
 PASSWORD=#PWD
 # Keycloak auth details 
-AUTH_URL=https://dev.oidc.gov.bc.ca/auth/realms
-REALM=dfmlcg7z
-CLIENT_ID=invasives-bc
\ No newline at end of file
+AUTH_URL=https://loginproxy.gov.bc.ca/auth/realms
```
Comment (norrisng-bc) [inline loadTest/app/env.example]: Old `AUTH_URL` is Silver DEV. Is the updated value supposed to be Gold DEV or PROD?

Comment (davidclaveau) [reply loadTest/app/env.example]: Definitely thought we caught that when we merged into the dev branch. I think Gold DEV is correct.

**Review by norrisng-bc — APPROVED** [review]

**Pull request merged**